src/init/init.js

```diff
@@ -96,16 +95,6 @@ const updateArtifacts = async (folder) => {
   await copyFolderRecursiveSync(fileSource, folder);
 };
 
-/**
- * generates a .gitignore file based on the content of
- * recommended-gitignore.txt
- * @param {string} folder - folder to add .gitignore
- */
-const generateGitIgnore = async (folder) => {
```

Comment:
I don't remember why we added this specifically @marc0olo as there is already a .gitignore included in the artifacts

Reply:
I think there was no gitignore included when I initialized a project back then. we definitely want to avoid users committing `node_modules` folder by accident 😅

Reply:
there is and there has been since 2021 according to git history